pyproject.toml

```diff
@@ -0,0 +1,63 @@
+[project]
+name = "sphinx-intl"
+dynamic = ['version']
+authors = [
+    { name = "Takayuki SHIMIZUKAWA", email = "shimizukawa@gmail.com" }
+]
+description = "Sphinx utility that make it easy to translate and to apply translation."
+readme = "README.rst"
+requires-python = ">=3.7"
+license = {file = "LICENSE"}
+dependencies = [
+    "setuptools",
+    "click",
+    "babel",
+    "sphinx",
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Environment :: Console",
+    "License :: OSI Approved :: BSD License",
+    "Topic :: Documentation",
+    "Topic :: Documentation :: Sphinx",
+    "Topic :: Software Development",
+    "Topic :: Software Development :: Documentation",
+    "Topic :: Text Processing",
+    "Topic :: Text Processing :: General",
+    "Topic :: Utilities",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Framework :: Sphinx",
+]
+
+[project.optional-dependencies]
+test = [
+    "pytest",
+    "mock",
+    "six",
+]
+
+[project.urls]
+homepage = "https://github.com/sphinx-doc/sphinx-intl"
+documentation = "https://sphinx-intl.readthedocs.io"
```

Comment:
You may want to apply case on these entries (e.g. `homepage` -> `Homepage`), since PyPI will use the key value for the text portion of the link.

Reply:
Thanks for the suggestion. Applied.